### PR TITLE
Feature/threshold ranges

### PR DESCRIPTION
**Pull request by pdorschner**
With this feature it is possible to specify thresholds as ranges. See [Nagios guidelines - Threshold and Ranges](https://nagios-plugins.org/doc/guidelines.html)

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -6,9 +6,9 @@ import (
 )
 
 var (
-	version = "0.1.1"
-	commit  = "b344608"
-	date    = "20.04.2022"
+	version = "0.1.2"
```
Comment (martialblog) [inline main.go:9]: Personally, I thing this merits a 0.2.0. 

Also the Date needs adjustment.

**Review by martialblog — CHANGES_REQUESTED** [review]
Looks good. 

Just the main.go needs some updates

File: main.go (hunk 1 above)
Comment (pdorschner) [reply main.go:9]: You are right, this PR adds an "new" functionality, so the minor version has to be raised.

The date ist already adjusted

Comment (martialblog) [reply main.go]: The date is still "25.04.2022"

**Review by martialblog — APPROVED** [review]
:+1:

**Pull request merged**